templates/interface/RedHat.erb

```diff
@@ -83,7 +83,6 @@ IPV6_ADDR="<%= @ipv6addr %>"
 IPV6_DEFAULTGW="<%= @ipv6_defaultgw %>"
 <% end -%>
 <% end -%>
```

Comment:
Never mind, just did it:
https://github.com/example42/puppet-network/commit/1939996fca974641527c75b57e01bf56628957f5